app/parsers/bulkrax/csv_parser.rb

```diff
@@ -74,8 +74,22 @@ def create_collections
       collections.each_with_index do |collection, index|
         next if collection.blank?
         break if records.find_index(collection).present? && limit_reached?(limit, records.find_index(collection))
+        ActiveSupport::Deprecation.warn(
+          'Creating Collections using the collection_field_mapping will no longer be supported as of Bulkrax version 3.0.' \
+          ' Please configure Bulkrax to use related_parents_field_mapping and related_children_field_mapping instead.'
+        )
+
+        metadata = if collection.key?(:title) && collection.keys.count == 1
+          {
+            title: [collection[:title]],
+            source_identifier: [collection[:title]],
```

Comment:
Look up dynamic `source_identifier` from `field_mappings`

Reply:
Actually, I believe this is [supposed to be work_identifier](https://github.com/samvera-labs/bulkrax/blob/v1.0.0/app/parsers/bulkrax/csv_parser.rb#L61)